### PR TITLE
Order summary: digital subscriptions

### PR DESCRIPTION
**Pull request by lucymonie**
## Why are you doing this?
This is the first step towards the new checkout rollout, using the design system 🎉 

Details and a link to the design are in the [Trello Card](https://trello.com/c/06UjK9kY/2935-checkouts-order-summary). The card assumes that this feature will be rolled out to all checkouts. As I've had to build a new component (because of using the design system) it was straightforward to leave the existing one in the other checkouts and just add this to digital. It should be uncomplicated to roll this out to the other checkouts from here if this is the preferred outcome.

## Changes
New order summary, per the design. 

## Accessibility test checklist
 - [x] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
    The text in the list that says 'So you'll pay £5.99/month for 3 months, then £11.99/month' on the desktop version doesn't make a lot of sense when read out by the screen reader. 
 - [x] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
   This isn't highly relevant as it doesn't have input fields or buttons
 - [x] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)
    Since this uses design system components, we can assume it meets an AA standard for colour contrast.

## Screenshots: mobile

Screen Shot 2020-03-30 at 19 02 59

## Screenshots: tablet

Screen Shot 2020-03-30 at 19 03 24

## Screenshots: desktop

Screen Shot 2020-03-30 at 19 04 01

### CODE REVIEW

File: support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import * as styles from './endSummaryStyles';
+
+const Dot = () => <div css={styles.dot} />;
+
+function EndSummary(props: {description: string}) {
+  return (
+    <ul css={styles.list}>
+      <li css={styles.listMain}><Dot />14-day free trial</li>
+      <li css={styles.listMain}>
+        <span><Dot />First 3 months half price</span>
```
Comment (rupertbates) [inline support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx]: This should copy come from the promotion not be hard coded

**Comment by johnduffell** [comment]
> The text in the list that says 'So you'll pay £5.99/month for 3 months, then £11.99/month' on the desktop version doesn't make a lot of sense when read out by the screen reader.

Good work spotting that. - Is that something we can/should fix or just an observation?

File: support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx (hunk 1 above)
Comment (lucymonie) [reply support-frontend/assets/components/subscriptionCheckouts/endSummary/endSummary.jsx]: True! Thanks Rupert

**Comment by lucymonie** [comment]
> > The text in the list that says 'So you'll pay £5.99/month for 3 months, then £11.99/month' on the desktop version doesn't make a lot of sense when read out by the screen reader.
> 
> Good work spotting that. - Is that something we can/should fix or just an observation?

Yes, I think we should address this. I have mentioned it to Gus, and hope that we can have a think about our wording more broadly. Probably not something we'll fix at this stage as this wording is right across subscriptions at the moment.

**Pull request merged**